### PR TITLE
Add version 1.15 of tienvx/mbt-bundle

### PR DESCRIPTION
**Pull request by tienvx**
| Q             | A
| ------------- | ---
| License       | MIT

This bundle now only support Symfony 5.0

### CODE REVIEW

File: tienvx/mbt-bundle/1.15/config/packages/tienvx_mbt.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+imports:
+    - { resource: dev/models/*.yaml }
+    - { resource: dev/predefined-cases/*.yaml }
+
+tienvx_mbt:
+#    max_steps: 300
+#    transition_coverage: 100
+#    place_coverage: 100
+#    default_bug_title: 'New bug'
```
Comment (Nyholm) [inline tienvx/mbt-bundle/1.15/config/packages/tienvx_mbt.yaml]: No need to have config in comments like this. Rather give a link to your documentation.

**Comment by Nyholm** [comment]
Friendly ping

**Comment by tienvx** [comment]
Hey @Nyholm. I'm sorry I can't update this PR sooner because I'm quite busy recently. I'm on Tet holiday so I can't update this PR. I'll update at the end of the holiday.

I'm sure about dropping Symfony 4.4, because I decide to depend on Symfony Notifier which is only available on Symfony 5.0. Also, this bundle is not expected to be installed with other bundles/libraries that use Symfony 4.4.

File: tienvx/mbt-bundle/1.15/config/packages/tienvx_mbt.yaml (hunk 1 above)
Comment (tienvx) [reply tienvx/mbt-bundle/1.15/config/packages/tienvx_mbt.yaml]: I simply remove the comments.

**Comment by tienvx** [comment]
Thank you

**Pull request merged**